client/packages/invoices/src/useDisableVvmRows.tsx

```diff
@@ -1,16 +1,41 @@
 import { useEffect } from 'react';
-import { useTableStore } from '@openmsupply-client/common';
+import {
+  PreferenceKey,
+  usePreference,
+  useTableStore,
+} from '@openmsupply-client/common';
 import { DraftStockOutLineFragment } from './StockOut';
 
 type RowsType = DraftStockOutLineFragment | DraftStockOutLineFragment;
 
-export const useDisableVvmRows = (rows: RowsType[]): void => {
+interface UseDisableVvmRowsProps {
+  rows: RowsType[];
+  isVaccine?: boolean;
+}
+
+export const useDisableVvmRows = ({
+  rows,
+  isVaccine,
+}: UseDisableVvmRowsProps): void => {
   const { setDisabledRows } = useTableStore();
+  const { data: prefs } = usePreference(
+    PreferenceKey.SortByVvmStatusThenExpiry,
+    PreferenceKey.ManageVvmStatusForStock
+  );
+
+  const shouldDisableVvmRows =
+    (prefs?.manageVvmStatusForStock || prefs?.sortByVvmStatusThenExpiry) &&
```

Comment:
I don't think the sort pref should be disabling the row...

Reply:
That makes sense, main reason I added was due to the thinking that if it's on then VVM is kicking in for the app. At least basing that from other places that have implemented same thing e.g. Columns for OutboundShipment's detail view

Reply:
I've removed it from there in my current PR 👀